custom-words.txt

```diff
@@ -1812,6 +1814,7 @@ scalesets
 Scame
 scanability
 scanrulesets
+scanstatus
```

Comment:
I cannot find "scanstatus" in your json. Is it needed here?

Reply:
Oh it must have been removed, thanks